src/zip.js

```diff
@@ -23,7 +23,7 @@ module.exports = function(gj, options) {
                     layers.file(fileName + '.shp', files.shp.buffer, { binary: true });
                     layers.file(fileName + '.shx', files.shx.buffer, { binary: true });
                     layers.file(fileName + '.dbf', files.dbf.buffer, { binary: true });
-                    layers.file(fileName + '.prj', prj);
+                    layers.file(fileName + '.prj', prj(options.wkt));
```

Comment:
```suggestion
                    layers.file(fileName + '.prj', options.wkt || prj);
```

Would this be easier than altering prj.js?

Reply:
made the change, thanks for the feedback!